ComposeSamples/ComposeGallery/src/main/java/com/microsoft/device/display/samples/composegallery/HomePage.kt

```diff
@@ -70,17 +79,78 @@ fun Home(viewModel: AppStateViewModel, windowInfoRep: WindowInfoRepository) {
 @Composable
 fun SetupUI(isDualMode: Boolean) {
     val models = DataProvider.imageModels
+    val imageSelectionLiveData = appStateViewModel.imageSelectionLiveData
+    val selectedIndex = imageSelectionLiveData.observeAsState(initial = 0).value
+
+    TwoPaneLayout(
+        paneMode = TwoPaneMode.TwoPane,
+        pane1 = { ShowList(models, isDualMode) },
+        pane2 = { ShowDetail(models, isDualMode, selectedIndex) }
+    )
+}
+
+@Composable
+private fun ShowWithTopBar(content: @Composable () -> Unit, actions: @Composable () -> Unit, title: String) {
```

Comment:
Good abstraction! Do you think it is a good idea to update TwoPaneLayout to support the usage inside the AppTopBar?

Reply:
I think that would be worth looking into! It may be good to leave multiple options available, in case developers want app bars present only in one pane or only in certain scenarios, but I think it would be useful to have a way to specify TopAppBar behavior with the TwoPaneLayout 😄